src/Swipeable.js

```diff
@@ -30,7 +31,8 @@ const Swipeable = React.createClass({
     return {
       flickThreshold: 0.6,
       delta: 10,
-      preventDefaultTouchmoveEvent: true
+      preventDefaultTouchmoveEvent: true,
+      component: 'div'
```

Comment:
we can might as well make it accept a React component, there's no harm in allowing it.